ambry-api/src/main/java/com/github/ambry/config/StatsManagerConfig.java

```diff
@@ -37,70 +37,70 @@ public class StatsManagerConfig {
    */
   @Config(STATS_OUTPUT_FILE_PATH)
   @Default("/tmp/stats_output.json")
-  public final String outputFilePath;
+  public final String statsOutputFilePath;
```

Comment:
The config name in close source should match with "stats.enable.mysql.report" and right now we should enable it. What you changed is just the variable name and used in open source only. I feel like this won't cause the issue.

Reply:
Tested this and changing the variable name fixed the issue. I was able to deploy the server on DR node.

Reply:
Will rebase and retest